app/build.gradle

```diff
@@ -103,11 +104,38 @@ buildscript {
     ext.assertj_version = '3.9.0'
     ext.constraint_layout_version = '1.1.3'
 }
+//Glide
+repositories {
+    mavenCentral()
```

Comment:
This is in the other build.gradle file. y think that we can remove this. Let me know if this is working.